projects/media-viewer/src/lib/comments-summary/comments-summary.component.html

```diff
@@ -1,7 +1,7 @@
-<div class="modal" (click)="onClose()">
-  <div class="modal-content govuk-width-container clearfix" (click)="$event.stopPropagation()">
+<div id="modal-background" class="modal" (click)="onClose()">
```

Comment:
The changes to this file will be used in another PR - it will allow the e2e tests to "click" on stuff, for the comment-summary feature